website/src/blogs/install-wordpress-using-dynamic-nfs-provisioner.md

```diff
@@ -0,0 +1,217 @@
+
+  #### Add helm repo for WordPress
+
+  - Use the below command to add helm repo for WordPress
+    ```sh
+    helm repo add bitnami https://charts.bitnami.com/bitnami
+    ```
+  - Once repo has been added successfully, update helm repo using the following command
+    ```sh
+    helm repo update
+    ```
+
+  #### Installing WordPress
+  Once the repo has been updated successfully, we can install WordPress referring to dynamic NFS provisioner storageclass. In this case storageclass will be openebs-kernel-nfs, if you have different storageclass then update the value of --set persistence.storageClass accordingly. WordPress also accepts various other parameters, for more information click [here](https://github.com/bitnami/charts/tree/master/bitnami/wordpress#parameters)
+
+  ```sh
+  helm install my-release -n wordpress --create-namespace \
+    --set wordpressUsername=admin \
+    --set wordpressPassword=password \
+    --set mariadb.auth.rootPassword=secretpassword \
+    --set mariadb.primary.persistence.enabled=true \
+    --set mariadb.primary.persistence.storageClass=openebs-hostpath \
+    --set mariadb.primary.persistence.accessModes={ReadWriteOnce} \
+    --set persistence.storageClass=openebs-kernel-nfs \
+    --set persistence.accessModes={ReadWriteMany} \
+    --set volumePermissions.enabled=true \
+    --set autoscaling.enabled=true \
+    --set autoscaling.minReplicas=2 \
+    --set autoscaling.maxReplicas=6 \
+    --set autoscaling.targetCPU=80 \
+    bitnami/wordpress
+  ```
+
+  Above helm installation should create following resources in the cluster:
+
+  - Two WordPress application pod instances with RWX persistent volume.
+  - One Mariadb application pod instance with RWO persistent volume.
+
+  Results of helm wordpress installation:
+  ```sh
+  kubectl get po -n wordpress
+  NAME                                    READY   STATUS    RESTARTS   AGE
+  my-release-mariadb-0                    1/1     Running   0          2m34s
+  my-release-wordpress-85dc688bb9-24jx2   1/1     Running   0          2m34s
+  my-release-wordpress-85dc688bb9-wkfpd   1/1     Running   1          2m18s
+  ```
+
+  Once the installation process is succeeded, follow the steps mentioned in output of _helm install_ command to access WordPress from your browser:
+
+  ![WordPress login page](./../../public/images/blog/install-wordpress-using-dynamic-nfs-provisioner-login-page.png)
```

Comment:
For images we can give the image link (the relative path) like this.
example:- 
![GitLab server](/images/blog/openebs-control-and-data-plane.png)

Reply:
```suggestion
  ![WordPress login page](/images/blog/install-wordpress-using-dynamic-nfs-provisioner-login-page.png)
```

Reply:
Updated as suggested